runelite-client/src/main/java/net/runelite/client/plugins/itemstats/special/CasteWarsBandage.java

```diff
@@ -0,0 +1,87 @@
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.itemstats.special;
+
+import com.google.common.collect.ImmutableSet;
+import lombok.RequiredArgsConstructor;
+import net.runelite.api.*;
+import net.runelite.client.plugins.itemstats.Effect;
+import net.runelite.client.plugins.itemstats.SimpleStatBoost;
+import net.runelite.client.plugins.itemstats.StatChange;
+import net.runelite.client.plugins.itemstats.StatsChanges;
+
+import java.util.Comparator;
+import java.util.stream.Stream;
+
+import static net.runelite.client.plugins.itemstats.Builders.heal;
+import static net.runelite.client.plugins.itemstats.Builders.perc;
+import static net.runelite.client.plugins.itemstats.stats.Stats.HITPOINTS;
+import static net.runelite.client.plugins.itemstats.stats.Stats.RUN_ENERGY;
+
+@RequiredArgsConstructor
+public class CasteWarsBandage implements Effect
+{
+	private static final ImmutableSet<Integer> BRACELETS = ImmutableSet.of(
+			ItemID.CASTLE_WARS_BRACELET1, ItemID.CASTLE_WARS_BRACELET2, ItemID.CASTLE_WARS_BRACELET3
+	);
+	private static final int GLOVES_SLOT_ID = EquipmentInventorySlot.GLOVES.getSlotIdx();
+	private static final double BASE_HP_PERC = .10;
+	private static final double BRACELET_HP_PERC = .50;
+	private static final SimpleStatBoost RUN_ENERGY_BOOST = heal(RUN_ENERGY, 30);
+
+	@Override
+	public StatsChanges calculate(Client client)
+	{
+		final StatsChanges changes = new StatsChanges(2);
+		final ItemContainer equipmentContainer = client.getItemContainer(InventoryID.EQUIPMENT);
+		final double percH = hasBracelet(equipmentContainer) ? BRACELET_HP_PERC : BASE_HP_PERC;
+		final SimpleStatBoost hitPoints = heal(HITPOINTS, perc(percH, 0));
+
+		changes.setStatChanges(
+				Stream.concat(
+				Stream.of(hitPoints.effect(client)),
+				Stream.of(RUN_ENERGY_BOOST.effect(client))
```

Comment:
Wouldn't `Stream.of(hitPoints.effect(client), RUN_ENERGY_BOOST.effect(client))` work instead? Is there a reason you are creating two separate streams then combining them?